bot/exts/evergreen/status_codes.py

```diff
@@ -0,0 +1,61 @@
+from http import HTTPStatus
+
+import discord
+from discord.ext import commands
+
+
+class StatusCats(commands.Cog):
+    """Commands that give HTTP statuses described and visualized by cats and dogs."""
+
+    def __init__(self, bot: commands.Bot):
+        self.bot = bot
+
+    @commands.group(name="http_status", aliases=("status", "httpstatus"))
+    async def http_status_group(self, ctx: commands.Context) -> None:
+        """Group containing dog and cat http status code commands."""
+        if not ctx.invoked_subcommand:
+            await ctx.send_help(ctx.command)
+
+    @http_status_group.command(aliases=['cat'])
+    async def http_cat(self, ctx: commands.Context, code: int) -> None:
+        """Sends an embed with an image of a cat, portraying the status code."""
+        embed = discord.Embed(title=f'**Status: {code}**')
+
+        try:
+            HTTPStatus(code)
+
+        except ValueError:
+            embed.set_footer(text='Inputted status code does not exist.')
+
+        else:
+            embed.set_image(url=f'https://http.cat/{code}.jpg')
```

Comment:
Could do: `http.cat` raises a 404 on some status codes that `HTTPStatus` says is a valid code, so we could handle those in a similar way to what you have done for the dog command.